### PR TITLE
feat(theme): Add some attributes for Tailwind

### PR DESCRIPTION
**Pull request by thenods**
Add some options for `TailwindCSS`

## Description
1. For `TailwindCSS`, added some options, `Font Family`, `Font Size`, and `Font Weight`

## Motivation and Context
image

FYI, `Font Family` might be changed.

## How Has This Been Tested?
1. Added className for the options that I added into the existing HTML Element
2. Check whether it's working

## Types of Changes

- [ ] Bug fix (non-breaking change that fixes an issue)
- [x] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

- [x] My code follows the code style of this project.

### CODE REVIEW

File: tailwind.config.js [hunk 1]
```diff
@@ -17,6 +17,21 @@ module.exports = {
       },
       black: '#1c1a1a',
     },
+    fontFamily: {
```
Comment (thenods) [inline tailwind.config.js:20]: https://tailwindcss.com/docs/font-family

File: tailwind.config.js [hunk 2]
```diff
@@ -17,6 +17,21 @@ module.exports = {
...
+      body: ['Apple SD Gothic Neo'],
+    },
+    fontSize: {
```
Comment (thenods) [inline tailwind.config.js:23]: https://tailwindcss.com/docs/font-size

File: tailwind.config.js [hunk 3]
```diff
@@ -17,6 +17,21 @@ module.exports = {
...
+      xs: '.75rem',
+      base: '0.9375rem',
+      lg: '1.125rem',
+      xl: '1.3125rem',
+      '2xl': '1.9375rem',
+    },
+    fontWeight: {
```
Comment (thenods) [inline tailwind.config.js:29]: https://tailwindcss.com/docs/font-weight

File: tailwind.config.js (hunk 1 above)
Comment (WDever) [reply tailwind.config.js:20]: How about just change `sans` properties? I think it's more common.

**Review by WDever — DISMISSED** [review]
LGTM ><

**Review by WDever — APPROVED** [review]
LGTM ><

**Pull request merged**